lib/matplotlib/font_manager.py

```diff
@@ -568,7 +544,7 @@ def createFontList(fontfiles, fontext='ttf'):
             except UnicodeError:
                 _log.info("Cannot handle unicode filenames")
                 continue
-            except IOError:
+            except OSError:
                 _log.info("IO error - cannot open font file %s", fpath)
```

Comment:
If its now an `OSError` should this message be changed, or is it obvious enough?

Reply:
Well, from the POV of the user it is indeed an input/output error... (and IOError == OSError anyways on Py3)